package.json

```diff
@@ -72,6 +73,8 @@
     "warning": "^3.0.0"
   },
   "devDependencies": {
+    "@types/enzyme": "^2.8.4",
+    "@types/react": "^15.0.39",
```

Comment:
These shouldn't be dev dependencies. They are needed for the typings to function.